### PR TITLE
Record player

### PR DESCRIPTION
**Pull request by MarkFootsy**
Updated Record Player to spin via triggers

### CODE REVIEW

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project. */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.RecordPlayerSubsystem;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
```
Comment (dawsonspub) [inline src/main/java/frc/robot/commands/SpinRecordPlayer.java]: remove unused import :)

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj.Joystick;
```
Comment (dawsonjeffreyd) [inline src/main/java/frc/robot/commands/SpinRecordPlayer.java]: remove unused import :)

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java (hunk 1 above)
Comment (dawsonjeffreyd) [inline src/main/java/frc/robot/commands/SpinRecordPlayer.java]: remove unused import :)

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java [hunk 3]
```diff
@@ -8,14 +8,16 @@
 package frc.robot.subsystems;
 
 import java.util.logging.Logger;
-import edu.wpi.first.wpilibj.VictorSP;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants.MotorPorts;
+import frc.robot.Constants;
+
```
Comment (violingirl) [inline src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: tsk..tsk...tsk. How dare you have an extra space.

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import frc.robot.Constants;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+public class RecordPlayerSubsystem extends SubsystemBase
+{
+   private TalonSRX recordPlayerMotor = new TalonSRX(Constants.CANID.recordPlayer);
+
+   /**
+    * Creates a new DriveSubsystem.
```
Comment (dawsonjeffreyd) [inline src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: update comment

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    */
+   public RecordPlayerSubsystem()
+   {
+      stopMotor();
+   }
+
+   /**
+    * Sets motor power for the Record Player Spinner motor
+    * 
+    * @param targetPower Accepts a power level between -1 to 1
```
Comment (dawsonjeffreyd) [inline src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: update parameter name

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java [hunk 6]
```diff
@@ -21,16 +22,32 @@
 {
    private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
    private final RecordPlayerSubsystem powerRecordPlayer;
+   private final RobotContainer m_robotContainer;
    private final double m_motorPower;
-   public SpinRecordPlayer(RecordPlayerSubsystem subsystem, double motorPower)
+
```
Comment (violingirl) [inline src/main/java/frc/robot/commands/SpinRecordPlayer.java:17]: The spaces strike again...

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java (hunk 3 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: checked in vs code, my word is superior, bwa ha ha

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java (hunk 4 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: fixed. yippee

File: src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java (hunk 5 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/subsystems/RecordPlayerSubsystem.java]: fixed as RecordPlayerPower

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java (hunk 1 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/commands/SpinRecordPlayer.java]: fixed

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java (hunk 2 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/commands/SpinRecordPlayer.java]: fixed

File: src/main/java/frc/robot/commands/SpinRecordPlayer.java (hunk 6 above)
Comment (MarkFootsy) [reply src/main/java/frc/robot/commands/SpinRecordPlayer.java:17]: i'll have you know mi padre TOLD me to put those there. but whatever

**Review by dawsonjeffreyd — APPROVED** [review]
Looks good to me

**Comment by dawsonjeffreyd** [comment]
Merging Isaac's record player code into Master

**Pull request merged**